MoltenVK/MoltenVK/Utility/MVKFoundation.h

```diff
@@ -24,6 +24,7 @@
 #include "mvk_vulkan.h"
 #include <algorithm>
 #include <string>
+#include <cassert>
```

Comment:
The assert later in this file fails to compile on Xcode-beta 12 without the inclusion of this header.